src/inverseRobot.js

```diff
@@ -24,24 +24,16 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  const obj = {};
-  const keys = Object.keys(robot);
+  const inverse = {};
 
-  for (let i = 0; i <= keys.length; i++) {
-    for (let j = i + 1; j <= keys.length; j++) {
-      if (robot[keys[i]] === robot[keys[j]]) {
-        return null;
-      }
+  for (const key in robot) {
```

Comment:
```suggestion
  for (const [key, value] of Object.entries(robot)) {
```

You can go even further and destructure robot's entries here to have both key and value inside your loop)